packages/client/src/v2-events/features/events/actions/declare/Pages.tsx

```diff
@@ -93,7 +93,9 @@ export function Pages() {
       onSaveAndExit={async () =>
         handleSaveAndExit(() => {
           drafts.submitLocalDraft()
-          goToHome()
+          searchParams.workqueue
```

Comment:
I want to make sure that we want to hold the workqueue state in the search params, instead of just in react state for example?